tox.ini

```diff
@@ -4,6 +4,7 @@ envlist = py27,py36,py37,py38,stylecheck
 [testenv]
 deps =
     pytest
+    flask<2.3.0 # For "before_first_request"
```

Comment:
Can you further clarify why this is needed?

We can't be pinning Flask to an old version unless it's a super temp pin until Flask drops a bugfix release... but Flask is now on `3.0.0` so methinks we'd want to test with that...

Reply:
`test/basic_app.py` still uses the `before_first_request` decorator which was [removed in Flask 2.3.0](https://flask.palletsprojects.com/en/2.3.x/changes/#version-2-3-0).
Additional changes would be necessary to make the extension compatible with `3.0.0`. If desired, I can prepare an additional PR for that.

Reply:
This is no longer part of the PR now that a few other PR's were merged re: compatibility.